python/ray/serve/_private/application_state.py

```diff
@@ -1030,4 +1040,9 @@ def override_deployment_info(
         ):
             deployment.route_prefix = app_route_prefix
 
+    # When there is no deployment entries and app logging config is set,
+    # apply app logging config to all existing deployment logging config.
+    if not deployment_override_options and app_logging_config:
+        for deployment in list(deployment_infos.values()):
+            deployment.deployment_config.logging_config = app_logging_config
```

Comment:
I think this logic is not comprehensive. If there are `deployment_override_options`, but the list is not complete, then the deployments not in the list will not have their logging config overridden. Am I correct?

Reply:
Updated logic:
1. if application logging config is set, apply to all existing deployment config.
2. if deployment logging config is set, will rely on existing for loop to update them.